### PR TITLE
Bump PLONK to `0.6`

### LINKED ISSUE #33: Update plonk to `0.6`

### PR DESCRIPTION
**Pull request by vlopes11**
Resolves #33

### CODE REVIEW

File: tests/zk.rs [hunk 1]
```diff
@@ -0,0 +1,364 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+#![feature(once_cell)]
```
Comment (ZER0) [inline tests/zk.rs:7]: Do not use `once_cell` yet. Unless they changed really recently, it's an nightly experimental feature that is still marked as unstable.

It's okay to use night-only feature, but they should be marked as stable.

**Review by ZER0 — CHANGES_REQUESTED** [review]
LGTM, few nits but the major issue is the tests is using an unstable feature; we should avoid that.

**Review by CPerezz — COMMENTED** [review]
LGTM as soon as @ZER0 's nits have been addressed.

**Comment by vlopes11** [comment]
N/A

**Pull request closed without merging**